components/pixels/snapshot.rs

```diff
@@ -92,13 +92,15 @@ pub enum SnapshotData {
         #[conditional_malloc_size_of] Arc<IpcSharedMemory>,
         Range<usize>,
     ),
+    SharedMemoryVec(#[conditional_malloc_size_of] Arc<Vec<u8>>, Range<usize>),
```

Comment:
I would avoid the use of `SharedMemory` in the name for this as it's a bit confusing. How about `SharedVec`?